### DISCUSSION TITLE
it seems that architecture diagram in the official website didn't update correspondingly

### DISCUSSION THREAD

**Discussion by RedemptionC** [apache/dolphinscheduler#7306] [General]
In the docs of 2.0,1.3.9,1.3.1,they all use this one:
![1.3.0](https://dolphinscheduler.apache.org/img/architecture-1.3.0.jpg)
And its uri shows that it's the architecture of 1.3.0
is that right？

**Comment by RedemptionC** [comment]
@dailidong

**Reply by Lucaszlei** [reply]
> @dailidong

This is the architecture in version 1.3.X, and may be changed in 2.0

**Reply by RedemptionC** [reply]
> > @dailidong
> 
> This is the architecture in version 1.3.X, and may be changed in 2.0

Yes,I watched a video by @dailidong on archSummit
The architecture of DS have changed from 1.2 to 1.3.x to 2.0
I think this should be reflected in the official website's Doc,which will make it easier for people unfamiliar with DS to get started

**Reply by davidzollo** [reply]
good suggestion, you can submit a new PR to official repo `https://github.com/apache/dolphinscheduler-website`

**Reply by RedemptionC** [reply]
@dailidong 
But I don't have the other two diagram 🤣 Where can I get them?

**Reply by zhongjiajie** [reply]
@RedemptionC I would help you find architecture in 2.0. Feel free to ping me if you have any question in our documentation

**Reply by zhongjiajie** [reply]
@RedemptionC I find out our leading page have the laster architecture diagram, maybe you could use this picture and upgrade our documentation
![image](https://user-images.githubusercontent.com/15820530/148488055-a88e8b9f-a689-4cb7-bafa-5f9e48e56522.png)

**Reply by zhongjiajie** [reply]
> good suggestion, you can submit a new PR to official repo `https://github.com/apache/dolphinscheduler-website`

BTW  @RedemptionC, do you like to submit PR for this?

**Comment by kaori-seasons** [comment]
@dailidong  Hello, this is my first contribution. I noticed your sharing on [DataFunSummit](https://www.cnblogs.com/DolphinScheduler/p/15794799.html). It seems to help me understand the architecture of this project. I will redraw the architecture diagram of three versions to complete the work.

**Reply by davidzollo** [reply]
> @dailidong Hello, this is my first contribution. I noticed your sharing on [DataFunSummit](https://www.cnblogs.com/DolphinScheduler/p/15794799.html). It seems to help me understand the architecture of this project. I will redraw the architecture diagram of three versions to complete the work.

thanks for the contribution, here is the newest Architecture of DolphinScheduler
![image](https://user-images.githubusercontent.com/15833811/171451616-dd0bb02a-b951-4f08-8b25-220aa91f9ebd.png)

by the way, are you interested in reviewing the official docs?
